ydb/core/tx/schemeshard/ut_helpers/ls_checks.h

```diff
@@ -23,7 +23,26 @@ struct TPathVersion {
     TPathId PathId = TPathId();
     ui64 Version = Max<ui64>();
 };
-using TApplyIf = TVector<TPathVersion>;
+
+struct TApplyIfUnit {
+    std::optional<TPathId> PathId;
+    std::optional<ui64> Version;
+    std::optional<std::vector<NKikimrSchemeOp::EPathType>> AllowedPathTypes;
```

Comment:
Why the use of `std::optional`s here?
`TPathVersion` already had well defined "unset" state. `PathTypes` -- the same.